python/test/t_Gibbs_std.py

```diff
@@ -213,7 +211,7 @@ def getParameter(self):
 # 3) a RandomWalkMetropolisHastings with average acceptance probability 1/2
 # If 1) is selected or 3) is selected and the proposal is rejected, the chain does not move
 # This happens with probability 1/3 + 1/3 * 1/2 = 1/2.
-sample = gibbs.getSample(10000)
+sample = gibbs.getSample(2000)
 diffs = sample[1:] - sample[:-1]
 zeros = ot.Point(4)
 null_diffs = [point == zeros for point in diffs]
```

Comment:
I cannot reach this with GitHub, but line 219 is too harsh with a sample of size 2000: use a relative tolerance of 0.04 instead of 0.02

Reply:
OK I'm on it